keanu-project/src/main/java/io/improbable/keanu/vertices/VertexDictionary.java

```diff
@@ -0,0 +1,18 @@
+package io.improbable.keanu.vertices;
+
+import java.util.Map;
+
+import io.improbable.keanu.tensor.Tensor;
+
+public interface VertexDictionary {
+
+    <V extends Vertex<? extends Tensor<?>>> V get(VertexLabel label);
```

Comment:
Does it have to return a tensor vertex?

Reply:
kinda yes (see comment above)

Reply:
Lets remove it for now and fix the bigger problem of `setValue`.

Reply:
done